Libraries/LibRegex/RegexMatcher.h

```diff
@@ -25,6 +25,7 @@ namespace Detail {
 struct Block {
     size_t start;
     size_t end;
+    StringView comment { "N/A"sv };
```

Comment:
I don't know if you intended to keep these debug prints, but if you do, they probably should go in their own commit.

Reply:
I do want to keep them.
Eh, sure, will move to a separate commit.